tests/testthat/test-target_sda.R

```diff
@@ -806,3 +806,17 @@ test_that("region_isos only has lowercase isos #398", {
     )
   )
 })
+
+test_that("region_isos only has lowercase isos #398", {
```

Comment:
seems like the test description is wrong, no?

Reply:
Ah yes, very much